frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -241,7 +238,7 @@ const NodesTableRow: React.FC<RowProps<NodeKind, GetNodeStatusExtensions>> = ({
   rowData,
 }) => {
   const { t } = useTranslation();
-  const metrics = useSelector(({ UI }) => UI.getIn(['metrics', 'node']));
+  const metrics = useSelector(({ UI }: RootState) => UI.getIn(['metrics', 'node']));
```

Comment:
Wherever we make these type definition updates, we should use the built-in `useSelector` types:
```suggestion
  const metrics = useSelector<RootState, string>(({ UI }) => UI.getIn(['metrics', 'node']));
```

Reply:
Thanks, @TheRealJon.  I made a pass at 'em.  Not 100% on all the return types (and even used `any` on a few that I noted with a comment).